### DISCUSSION TITLE
how to set video filters by default

### DISCUSSION THREAD

**Discussion by RUKNEG** [smplayer-dev/smplayer#926] [Q&A]
Version : 23.12.0 (révision 10207)
Utilise Qt 5.12.8 (compilé avec Qt 5.12.8)
Utilise MPV 0.32.0

I'll like to know how set videos filters by default, not only interlacing and video EQ.
Maybe that need to be put in "video player option", but can't find working syntax (given example by tooltip are outdated).

Thanks

**Comment by RUKNEG** [comment]
![sm-f](https://github.com/smplayer-dev/smplayer/assets/146966461/94419b26-0c8f-4941-9edc-b1e17ee2f2c0)

**Comment by smplayer-dev** [comment]
Add this line in preferences -> advanced:
`postprocessing true deblock true dering true gradfun true`

![Captura desde 2024-03-18 15-14-26](https://github.com/smplayer-dev/smplayer/assets/10128951/ff36f1e7-74c8-455d-aba7-1e9fe850b947)

**Reply by RUKNEG** [reply]
Thanks for tip, but where did you found this syntax?
rem: postprocessing can be automatically activated in preference, but it's the only one.

**Reply by smplayer-dev** [reply]
You can find the names of the filters in preferences -> keyboard and mouse. Add true to enable it or false to disable it.
